instagram/src/components/PostContainer/postcontainer.js

```diff
@@ -3,19 +3,43 @@ import CommentSection from '../CommentSection/commentsection'
 import styles from './postcontainer.css';
 import PropTypes from 'prop-types';
 
-const PostContainer = (props) =>{
+class PostContainer extends React.Component{
+  constructor(props){
+    super(props)
+    this.state = {
+      likes: this.props.dummyObj.likes
+    }
+  }
+  addLike = () =>{
+    this.setState({
+      likes : ++this.state.likes
```

Comment:
the `++` operator modifies state directly, which is bad practice in React. Use `this.state.likes + 1` instead